test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperRunnerTest.cs

```diff
@@ -10,6 +10,79 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
 {
     public class TagHelperRunnerTest
     {
+        public static TheoryData TagHelperOrderData
+        {
+            get
+            {
+                var tagHelper1000 = new OrderedTagHelper(1000);
+                var tagHelperMAX = new OrderedTagHelper(int.MaxValue);
+                var tagHelper0 = new OrderedTagHelper(0);
+                var tagHelperN1000 = new OrderedTagHelper(-1000);
+                var tagHelperMIN = new OrderedTagHelper(int.MinValue);
+
+                // tagHelperAddOrder, expectedTagHelperOrder
+                return new TheoryData<OrderedTagHelper[], OrderedTagHelper[]>
```

Comment:
why isn't this `TheoryData<int[], int[]>`?  this data cannot currently be used in tests running in parallel.